### PR TITLE
Added more information to parser errors

### PR DESCRIPTION
**Pull request by poshul**
I've added a couple of more specific errors for failure to wrap and failure to parse. These have helped with my debugging.

### CODE REVIEW

**Comment by uweschmitt** [comment]
Thanks for the contribution. I'm just reviewing pending merge request and wonder how the added boost core files relate to the improved error messages.

**Comment by poshul** [comment]
> 
> 
> Thanks for the contribution. I'm just reviewing pending merge request and wonder how the added boost core files relate to the improved error messages.

Those got dragged in from another PR I was working on. The upstream branch now uses them.

File: autowrap/PXDParser.py [hunk 1]
```diff
@@ -131,6 +132,10 @@ def parse_line_annotations(node, lines):
                     # they belong to the previous key
                     value = " " + f_
                     result[key] += value
+      except:
```
Comment (uweschmitt) [inline autowrap/PXDParser.py:135]: You could rephrase this as 
```
except Exception as e:
    raise ValueError("cannot parse '{}'".format(line)) from e
```

Comment (poshul) [reply autowrap/PXDParser.py]: Done.

**Comment by poshul** [comment]
@uweschmitt  anything else needed to merge this?
